docs/content/myst.md

```diff
@@ -159,7 +160,9 @@ For a list of directives that are available to you, there are three places to ch
    has a list of directives in the Python "docutils" module.
 3. This documentation has several additional directives that are specific to Jupyter Book.
 
-In some unusual cases, MyST-Parser may be incompatible with a certain role or directive.
+:::{admonition} What if it exists in rST but not MyST?
```

Comment:
@choldgraf we could also just use `:::{tip}` in this case right?

Reply:
I have been using

````
```{tip}
````

so perhaps `:::` is different.

Reply:
Yes with one exception, the `{tip}`, `{note}` etc admonitions don't let you specify a custom title (in those cases, the title is "tip", "note", etc). So this is a way to add your own title.

And ` ```{tip} ` should work the exact same way as `:::{tip}` now